amazon-codeguru-reviewer-sample-app-master/src/main/java/com/shipmentEvents/handlers/EventHandler.java

```diff
@@ -116,11 +116,11 @@ private void processShipmentUpdates(final LambdaLogger logger) throws Interrupte
     }
 
     private List<KeyVersion> processEventsInBucket(String bucketName, LambdaLogger logger, Map<String, Pair<Long, String>> latestStatusForTrackingNumber) {
-        final AmazonS3 s3Client = EventHandler.getS3Client();
-        logger.log("Processing Bucket: " + bucketName);
-
-        ObjectListing files = s3Client.listObjects(bucketName);
-        List<KeyVersion> filesProcessed = new ArrayList<DeleteObjectsRequest.KeyVersion>();
+            final AmazonS3 s3Client = EventHandler.getS3Client();
+            logger.log("Processing Bucket: " + bucketName);
+    
+            ObjectListing files = s3Client.listObjects(bucketName);
```

Comment:
### CodeGuru Findings
**Finding Title:** APIs that are not recommended were found.
**Description:** APIs that are not recommended were found. This could indicate a deeper problem in the code.
**Severity:** High